HACKING.md

```diff
@@ -59,6 +59,10 @@ export CONSUMER_ID=$(openssl x509 -in cert.pem -subject -nocert | cut -f3 -d" ")
 mosquitto_pub --host 127.0.0.1 --port 1883 --topic "yggdrasil/${CONSUMER_ID}/control/in" --message "{\"type\":\"command\",\"message_id\":\"$(uuidgen | tr -d '\n')\",\"version\":1,\"sent\":\"$(date --iso-8601=seconds --utc | tr -d '\n')\",\"content\":{\"command\":\"ping\"}}"
 ```
 
+echo '{"command":"cancel", "arguments":{"directive":"echo","messageID":"dbc8c1a7-8552-4b79-b36d-723aa68c25dc"}}' | go run ./cmd/yggctl generate control-message --type command - | pub -broker tcp://test.mosquitto.org:1883 -topic yggdrasil/$(hostname)/control/in
```

Comment:
Can you please add some description of this new commands? This change should be in another commit, because it looks that it is not related to the fix.

Reply:
Oh this is a wrong line, I'm not sure how it got there. Deleting it.